modules/shortlinks.php

```diff
@@ -85,5 +85,11 @@ function wpme_get_shortlink( $id = 0, $context = 'post', $allow_slugs = true ) {
 }
 
 function wpme_get_shortlink_handler( $shortlink, $id, $context, $allow_slugs ) {
-	return wpme_get_shortlink( $id, $context, $allow_slugs );
+	if ( $shortlink === false ) {	// respect / do not change the value from previous filter hooks
```

Comment:
Could you use Yoda conditions here, as per WordPress' coding standards?
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions

Reply:
I'm not a fan, lol, but sure - done. :)